typescript/opensearch/cwlogs_ingestion/resources/lambda/log_emitter/handler.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
+import json
+from random import randrange
+
+def log_emitter(event, context):
+    source = {}
+    id = str(randrange(10000))
+    source['id'] = id
+    source['timestamp'] = str(datetime.now())
+    source['message'] = 'Hello world'
```

Comment:
It may be nice to make this more interesting and look like some application log.